common/ClientRequestCleaner.js

```diff
@@ -0,0 +1,19 @@
+const RequestsCleaner = require('./RequestsCleaner');
+
+class ClientRequestsCleaner extends RequestsCleaner {
+	verifySignatures() {
+		if (!this.calls) {
+			return;
+		}
+		Object.keys(this.calls).forEach((procedure) => {
+			Object.keys(this.calls[procedure]).forEach((signature) => {
+				if (RequestsCleaner.isOutdated(signature, this.timeoutMs)) {
+					this.calls[procedure][signature].reject('RPC response timeout exceeded');
```

Comment:
Thank you very much for the suggestion. I've opened a separate issue to address it #21